tests/sentry/api/endpoints/test_auth_config.py

```diff
@@ -39,7 +39,7 @@ def test_superuser_is_not_redirected(self):
         response = self.client.get(self.path)
 
         assert response.status_code == 200
-        assert "nextUri" not in response.data
```

Comment:
As we are no longer granting superuser access upon login, we don't need to redirect the user back to sso login and they can continue with the normal flow